src/program.js

```diff
@@ -182,6 +182,10 @@ Example: $0 --help run.
           describe: 'Watch for file changes and re-build as needed',
           type: 'boolean',
         },
+        'extension': {
+          describe: 'Change the file extension from .zip to something else',
```

Comment:
To make it clear that the value should not include a dot, maybe add another sentence: "Example: --extension=xpi"